packages/wrangler/src/pages/deploy.tsx

```diff
@@ -337,9 +345,65 @@ export const Handler = async (args: PagesDeployArgs) => {
 		project_name: projectName,
 	});
 
-	logger.log(
-		`✨ Deployment complete! Take a peek over at ${deploymentResponse.url}`
-	);
+	let latestDeploymentStage: DeploymentStage | undefined;
+	let attempts = 0;
+
+	logger.log("🌎 Checking deployment status...");
```

Comment:
Maybe this should just be "Deploying..."? That's what we're actually doing while waiting, I don't think we need to surface our status checking to the user.

Reply:
✅